src/main/resources/travis/travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Whoops yeah. :) _Executables_ go under src/bin.

Speaking of binary files that shouldn't ~~bloat~~ go in the artifact, I'm not really sure where jesConf.tar.enc should go.

Can't find the discussion on the topic at the moment, but these two links show "the" conventional directory structure, plus how maven itself organizes under the src directory (including src/bin):

- https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html
- https://github.com/apache/maven/tree/maven-3.3.9/apache-maven/src